skimage/draw/draw.py

```diff
@@ -693,3 +693,51 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+def rect(origin,extent,shape = None):
+    """Generate coordinates of pixels within a nd-rectangle.
+
+    Parameters
+    ----------
+    origin, extent : tuple
+        origin and extent of the rectangle.
+    shape : tuple, optional
+        Image shape which is used to determine the maximum extent of output
+        pixel coordinates. This is useful for rectangles that exceed the image
+        size. If None, the bounds are determined from the bounds of
+        the rectangle (origin + extent).
+
+    Returns
+    -------
+    coords : sequence of ndarrays of int
```

Comment:
"sequence of ndarrays of int" doesn't sound quite right.  Perhaps: `(M, n) ndarray of int` or something similar.